### PR TITLE
Fix links in README; remove language from url-path

### PR DESCRIPTION
**Pull request by ketsappi**
## Description
Fix the links in the README.
No longer having the language in url-path as we are not using i18n solution anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Suomi.fi Design System
 
-Documentation website for the [Suomi.fi Design System](https://vrk-kpa.github.io/suomifi-design-system/fi/)
+Documentation website for the [Suomi.fi Design System](https://vrk-kpa.github.io/suomifi-design-system/)
```
Comment (valger11) [inline README.md:4]: https://vrk-kpa.github.io/suomifi-design-system/ is mentioned three times. Maybe 1 would be enough?

Comment (ketsappi) [reply README.md:4]: Yes, that is a good point. We could rewrite this README on that part.

**Review by LJKaski — APPROVED** [review]
This change will do for now. Let's rewrite the readme later.

File: README.md (hunk 1 above)
Comment (ketsappi) [reply README.md:4]: README will be rewritten at better time.

**Pull request merged**